app/policies/spree/retail/shopify/can_issue_refund_policy.rb

```diff
@@ -0,0 +1,44 @@
+require_dependency 'spree/retail/shopify'
+
+module Spree
+  module Retail
+    module Shopify
+      class CanIssueRefundPolicy
+        def initialize(transaction:, amount_to_credit:)
+          @amount_to_credit, @transaction = amount_to_credit, transaction
```

Comment:
Not a super fan of initialization iVar's like this due to it being somewhat tricky when 'quickly scanning'. Also doesn't Rubocop raise a little warning with these as well?